openshift/release/eventing.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+wget https://raw.githubusercontent.com/openshift/knative-eventing/release-next/openshift/release/knative-eventing-ci.yaml
```

Comment:
In theory there is a second file, but I guess you do not really need the `knative-eventing` bits, right ?

Reply:
If we don't use it at all, could we put a dummy file there?